pkg/monitortests/node/kubeletselinuxlabels/monitortest.go

```diff
@@ -0,0 +1,172 @@
+				WithAnnotation(monitorapi.AnnotationRoles, roles).
+				WithAnnotation(monitorapi.AnnotationPhase, "OperatingSystemUpdate")
+			intervals = append(intervals, nodeStateTracker.CloseIfOpenedInterval(nodeLocator, osUpdateState,
+				statetracker.SimpleInterval(monitorapi.SourceNodeState, monitorapi.Info, osUpdateMB),
+				event.From)...)
+			nodeStateTracker.OpenInterval(nodeLocator, rebootState, event.From)
+		case "Starting":
+			// Not ported, so we don't have a Source to check
+			mb := monitorapi.NewMessage().Reason(monitorapi.NodeUpdateReason).
+				HumanMessage(msgPhaseDrain).
+				WithAnnotation(monitorapi.AnnotationConstructed, monitorapi.ConstructionOwnerNodeLifecycle).
+				WithAnnotation(monitorapi.AnnotationRoles, roles).
+				WithAnnotation(monitorapi.AnnotationPhase, "Drain")
+			intervals = append(intervals, nodeStateTracker.CloseIfOpenedInterval(nodeLocator, drainState,
+				statetracker.SimpleInterval(monitorapi.SourceNodeState, monitorapi.Info, mb),
+				event.From)...)
+
+			osUpdateMB := monitorapi.NewMessage().Reason(monitorapi.NodeUpdateReason).
+				HumanMessage(msgPhaseOSUpdate).
+				WithAnnotation(monitorapi.AnnotationConstructed, monitorapi.ConstructionOwnerNodeLifecycle).
+				WithAnnotation(monitorapi.AnnotationRoles, roles).
+				WithAnnotation(monitorapi.AnnotationPhase, "OperatingSystemUpdate")
+			intervals = append(intervals, nodeStateTracker.CloseIfOpenedInterval(nodeLocator, osUpdateState,
+				statetracker.SimpleInterval(monitorapi.SourceNodeState, monitorapi.Info, osUpdateMB),
+				event.From)...)
+
+			rebootMB := monitorapi.NewMessage().Reason(monitorapi.NodeUpdateReason).
+				HumanMessage(msgPhaseReboot).
+				WithAnnotation(monitorapi.AnnotationConstructed, monitorapi.ConstructionOwnerNodeLifecycle).
+				WithAnnotation(monitorapi.AnnotationRoles, roles).
+				WithAnnotation(monitorapi.AnnotationPhase, "Reboot")
+			intervals = append(intervals, nodeStateTracker.CloseIfOpenedInterval(nodeLocator, rebootState,
+				statetracker.SimpleInterval(monitorapi.SourceNodeState, monitorapi.Info, rebootMB),
+				event.From)...)
+		}
+	}
+	// Close all node intervals left hanging open:
+	intervals = append(intervals, nodeStateTracker.CloseAllIntervals(locatorToMessageAnnotations, end)...)
+
+	return intervals, nil
+}
+
+func (lw *selinuxLabelWatcher) EvaluateTestsFromConstructedIntervals(ctx context.Context, finalIntervals monitorapi.Intervals) ([]*junitapi.JUnitTestCase, error) {
+	testCaseJunit := junitapi.JUnitTestCase{Name: testName}
+	kubeClient, err := kubernetes.NewForConfig(lw.adminRESTConfig)
+	if err != nil {
+		return []*junitapi.JUnitTestCase{&testCaseJunit}, err
+	}
+	nodes, err := kubeClient.CoreV1().Nodes().List(ctx, v1.ListOptions{})
+	if err != nil {
+		return []*junitapi.JUnitTestCase{&testCaseJunit}, err
+	}
+	for _, val := range nodes.Items {
+		// calls ps -Z -C kubelet
+		out, err := exutil.DebugNodeRetryWithOptionsAndChroot(lw.oc, val.Name, "default", "/bin/bash", "-c", "ps -Z -C kubelet")
```

Comment:
I suspect the ps -Z -C kubelet will need to be split out to:

"ps", "-Z", "-C", "kubelet")

Reply:
Hmm. But this might work because of the -c

Reply:
Right now when testing locally, I am getting a panic.